dts/arm/nxp/nxp_lpc55S2x_common.dtsi

```diff
@@ -267,6 +273,15 @@
 		offset-value-b = <10>;
 		#io-channel-cells = <1>;
 	};
+
+	usbhs: usbhs@144000 {
+		compatible = "nxp,mcux-usbd";
+		reg = <0x94000 0x1000>;
+		interrupts = <47 1>;
+		num-bidir-endpoints = <5>;
```

Comment:
According to the RM (Chapter 44), this should be 6. Can you please check and help update. I think the same change should happen for LPC55S69 as well.

Reply:
@mmahadevan108 , according to the RM, there is 6 endpoints, but 0 is for control.